src/js/game/LevelMVC/LevelView.js

```diff
@@ -2079,6 +2110,12 @@ module.exports = class LevelView {
       };
 
       const collectiblePosition = this.controller.levelModel.spritePositionToIndex(offset, new Position(sprite.x, sprite.y));
+
+      // Chests will put out a treasure, but not disappear, so you need to be able to pick up set item from further away than the normal 1 index
+      if (blockType === "heartofthesea" || blockType === "boat" || blockType === "prismarine" || blockType === "nautilus") {
+        collectibleDistance *= 2;
+      }
```

Comment:
This is already overridable via the `overrides` argument.  But I think it's actually not bad to have the item show up and not collect it — then you can actually see what it is, vs. it disappearing right away.